### PR TITLE
Feature/orca 780 Update Orca documentation for use with newer versions of Orca/Cumulus Core

### PR DESCRIPTION
**Pull request by rizbihassan**
## Summary of Changes

Addresses [ORCA-780: Update Orca documentation for use with newer versions of Orca/Cumulus Core](https://bugs.earthdata.nasa.gov/browse/ORCA-780)

## Changes

* Updated deployment-with-cumulus.md to run orca workflows.
* 
## Type of change

- [ x] Bug fix (non-breaking change which fixes an issue)
- [x ] This change requires a documentation update

## How Has This Been Tested?

Deployed orca webpage and checked the formatting.

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [ x] CHANGELOG.md
    - [x ] Development documentation
- [x ] My changes generate no new warnings
- [ x] My code has passed security scanning
    - [x ] Snyk
    - [ x] git-secrets

### CODE REVIEW

**Review by zroehrich — APPROVED** [review]

File: website/docs/developer/deployment-guide/deployment-with-cumulus.md [hunk 1]
```diff
@@ -413,7 +429,7 @@ the ingest workflow.
   }
 },
   "Type":"Task",
-  "Resource":"module.orca.orca_lambda_copy_to_archive_arn",
+  "Resource":"module.orca.orca_lambda_copy_to_archive_adapter.arn",
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deployment-with-cumulus.md:432]: Should be a variable that points to the cumulus adapter. May need to look at their TF stuff to provide the path

**Review by bhazuka — CHANGES_REQUESTED** [review]
A few confusing items to rectify

File: website/docs/developer/deployment-guide/deployment-with-cumulus.md (hunk 1 above)
Comment (rizbihassan) [reply website/docs/developer/deployment-guide/deployment-with-cumulus.md:432]: Will reference to `https://github.com/nasa/cumulus/blob/master/tf-modules/cumulus/outputs.tf#L86` for users to add the value.

File: website/docs/developer/deployment-guide/deployment-with-cumulus.md [hunk 2]
```diff
@@ -393,6 +404,11 @@ the ingest workflow.
 
 :::
 
+Since ORCA is decoupling from Cumulus starting in ORCA v8.0, users will now run the same [ORCA `copy_to_archive` workflow](https://github.com/nasa/cumulus-orca/tree/master/modules/workflows/OrcaCopyToArchiveWorkflow) but must need to update the existing workflow configuration to point to [copy_to_archive_adapter lambda](https://github.com/nasa/cumulus/tree/master/tasks/orca-copy-to-archive-adapter) (owned by Cumulus) which then runs our existing orca archive workflow.
```
[the remark this answers is not in the document]
Comment (rizbihassan) [reply website/docs/developer/deployment-guide/deployment-with-cumulus.md:407]: I did point to `https://github.com/nasa/cumulus/tree/master/tasks/orca-copy-to-archive-adapter` here. Not sure if I am missing anything else.

**Review by bhazuka — APPROVED** [review]
We should probably put a card in Jira to remove our proof of concept adapters in the future

**Comment by rizbihassan** [comment]
> We should probably put a card in Jira to remove our proof of concept adapters in the future

Created ORCA-783

**Pull request merged**